docs/configuration/configuration.md

```diff
@@ -61,6 +61,13 @@ global:
   # How frequently to evaluate rules.
   [ evaluation_interval: <duration> | default = 1m ]
 
+  # File to which PromQL queries are logged if <query_log_min_duration> is set.
```

Comment:
Why would the duration be required for logging to work?

Reply:
remove duration